docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
       dockerfile: Dockerfile
     command: >
       bash -c 'cd /app &&
-             yarn && 
+             yarn install --silent && 
```

Comment:
not so sure we want to do this here

Reply:
Completed - removed the `--silent` flag and now simply install the dependencies from a lock file. Thank you!